public/index.html

```diff
@@ -10,7 +10,7 @@
 
     <script src="https://cfpb.github.io/capital-framework/dist/static/js/main.min.js"></script>
     <!--[if IE 9]><!-->
-    <script src="https://cdn.polyfill.io/v2/polyfill.js?features=Intl.~locale.en"></script>
+    <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,Intl.~locale.en"></script>
```

Comment:
Can we call `polyfill.min.js` to save some bytes?

Reply:
Sure